samples/common/mcumgr_bt_ota_dfu/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if (CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_VALIDATION)
```

Comment:
Maybe you could also comment here that the network core child image configuration overlay is applied automatically in the main CMakelists.txt file defined in samples directory (it seems like part of this functionality, but it cannot be moved here because of the way how child image is introduced to the build system).

Reply:
CC: @mkapala-nordic, I think this should be mentioned in the UG documentation